### ISSUE TITLE
codeceptjs/puppeteer@14.4.1 fails on handleDownloads()

### ISSUE THREAD

**Issue by apydo** [Codeception/CodeceptJS#3341]
#### What are you trying to achieve?
Use handleDownloads() to download file in test with codeceptjs/puppeteer

#### What do you get instead?
On I.handleDownloads() there is the following error:
```
     this.page._client.send is not a function
      at Puppeteer.handleDownloads (node_modules/codeceptjs/lib/helper/Puppeteer.js:1072:30)
      at Step.run (node_modules/codeceptjs/lib/step.js:122:47)
      at /content/n/d3/node_modules/codeceptjs/lib/actor.js:134:23

```
### Details

* CodeceptJS version: 3.3.3
* NodeJS Version: 16.15.1
* Operating System: Linux (WSL2)
* puppeteer: 14.4.1
* Configuration file:
```js
{
  tests: ['./test/*.js', './test/testdir/*.js'],
  output: './test/output',
  helpers: {
    Puppeteer: {
      url: 'http://localhost:8080',
      show: true,
      windowSize: '1200x900',
      chrome: {
        args: ['--no-sandbox', '--window-size=1280,960'],
        defaultViewport: {
          width: 1280,
          height: 960
        }
      }
    },
    FileSystem: {},
    clickIfVisible: {
      require: './test/helpers/click-if-visible.js',
    },
  },
  include: {
    I: './steps_file.js'
  },
  bootstrap: null,
  mocha: {},
  name: 'LAP',
  plugins: {
    pauseOnFail: {},
    retryFailedStep: {
      enabled: true
    },
    screenshotOnFail: {
      enabled: true
    }
  }
}
```

My test worked fine and now it fails on 'I.handleDownloads() with the error shown above.
All I changed was the puppeteer version.

Everything is fine with puppeteer@14.3.0 (and previous versions).
Since puppeteer@14.4.0 (including current 14.4.1) the test always fails.

**Comment by solygen** [comment]
+1

**Comment by kospvn** [comment]
+1

**Comment by asnaseer-resilient** [comment]
I just got the same error with puppeteer@15.5.0

**Comment by kobenguyent** [comment]
As far as I know since 14.4.0, all private methods are not public so that explains why. Perhaps we should fix the Puppeteer helper then. Like there is a fix for #3370
